config/hsc/convertRepo.py

```diff
@@ -6,6 +6,7 @@
 # probably won't survive persisting the config.
 import lsst.obs.subaru.gen2to3
 
+config.instrument = "lsst.obs.subaru.HyperSuprimeCam"
```

Comment:
Why is this set in this config but the DECam equivalent doesn't set instrument?

Reply:
I hadn't noticed that DECam didn't because the command-line script code sets it.  I think I should probably set it in DECam in order to make the `Task` more usable directly (though that's true of a lot of the logic in the script, I think).

In any case, it's an example of the whole mess where you have to specify an instrument to get the obs-package overrides that then set the instrument.  I think that's one of the problems that would go away if we move `writeCuratedCalibrations` and instrument registration out of `ConvertRepoTask`, because those are the only reasons we need the `Instrument` there.